### PR TITLE
cinzel: init at 0-unstable-2020-07-22

### PR DESCRIPTION
**Pull request by marijanp**
## Description of changes
Unsure about the version, there is no release tag so I decided to use the revision.

https://fonts.google.com/specimen/Cinzel
https://github.com/NDISCOVER/Cinzel

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/ci/cinzel/package.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{ lib, stdenvNoCC, fetchFromGitHub, rename }:
+
+stdenvNoCC.mkDerivation {
+  pname = "cinzel";
+  version = "dd59849";
```
Comment (qubitnano) [inline pkgs/by-name/ci/cinzel/package.nix:5]:
```suggestion
  version = "0-unstable-2020-07-22";
```

Comment (marijanp) [reply pkgs/by-name/ci/cinzel/package.nix]: Thank you, updated it.

Comment (pluiedev) [reply pkgs/by-name/ci/cinzel/package.nix]: Also should update the PR title :p

File: pkgs/by-name/ci/cinzel/package.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+stdenvNoCC.mkDerivation {
+  pname = "cinzel";
+  version = "0-unstable-2020-07-22";
+
+  src = fetchFromGitHub {
+    owner = "NDISCOVER";
+    repo = "Cinzel";
+    rev = "dd598495b0fb2ad84270d5cc75d642d2f1e8eabf";
+    sha256 = "sha256-V3vSxe5eHN0BUplzmmBchzuV+gz36WfgZZezJ6NfaWg=";
```
Comment (pluiedev) [inline pkgs/by-name/ci/cinzel/package.nix:11]:
```suggestion
    hash = "sha256-V3vSxe5eHN0BUplzmmBchzuV+gz36WfgZZezJ6NfaWg=";
```

**Review by pluiedev — CHANGES_REQUESTED** [review]
Just one tiny nitpick. Also would be nice to add an update script via `nix-update-script`

```nix
  passthru.updateScript = nix-update-script {
    extraArgs = [ "--version=branch" ];
  }
```

File: pkgs/by-name/ci/cinzel/package.nix (hunk 1 above)
Comment (marijanp) [reply pkgs/by-name/ci/cinzel/package.nix]: Where did you get this version string from?

File: pkgs/by-name/ci/cinzel/package.nix (hunk 2 above)
Comment (marijanp) [reply pkgs/by-name/ci/cinzel/package.nix:11]: did that

File: pkgs/by-name/ci/cinzel/package.nix (hunk 1 above)
Comment (marijanp) [reply pkgs/by-name/ci/cinzel/package.nix]: I cannot find this proposed version string anywhere

**Comment by marijanp** [comment]
@pluiedev where can I find documentation on nix-update-script what does the argument do? Also the proposed version string does not exist, so I assume that your proposed passthru won't work as expected

**Comment by paparodeo** [comment]
>  Also the proposed version string does not exist, so I assume that your proposed passthru won't work as expected

Re version string: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

> If a package is a commit from a repository without a version assigned, then the version attribute should be the latest upstream version preceding that commit, followed by -unstable- and the date of the (fetched) commit. The date must be in "YYYY-MM-DD" format.
>
> Example: Given a project had its latest releases 2.2 in November 2021, and 3.0 in January 2022, a commit authored on March 15, 2022 for an upcoming bugfix release 2.2.1 would have version = "2.2-unstable-2022-03-15".
>
> If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be

**Comment by pluiedev** [comment]
> @pluiedev where can I find documentation on nix-update-script what does the argument do? Also the proposed version string does not exist, so I assume that your proposed passthru won't work as expected

nix-update-script is a wrapper around Mic92/nix-update, and all the arguments are documented there. `--version=branch` actually means 'latest commit of default branch', so it should work as intended

**Comment by drupol** [comment]
It looks like you accidentally mass-pinged a bunch of people, which are now subscribed
and getting notifications for everything in this pull request. Unfortunately, they
cannot be automatically unsubscribed from the issue (removing review request does not
unsubscribe), therefore development cannot continue in this pull request anymore.

Please create a new pull request, link back to this one and ping the
people actually involved in here over there. For the next time, remember to set your PR to draft status before rebasing. In draft status, you can preview the list of maintainers that are about to be requested for review, which allows you to sidestep this issue.

In order to avoid this in the future, there are instructions for how to properly
rebase between branches in our [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging).
Setting your pull request to draft prior to rebasing is strongly recommended.
In draft status, you can preview the list of people that are about to be requested
for review, which allows you to sidestep this issue.
This is not a bulletproof method, though, as OfBorg still does review requests even on draft PRs.

**Pull request closed without merging**